modules/api-compatibility-exceptions.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * rest_api/understanding-compatibility-guidelines.adoc
+
+[id="api-compatibility-exceptions_{context}"]
+= API compatibility exceptions
+
+The following are exceptions to compatibility in {product-title}:
+
+[discrete]
+[id="OS-file-system-modifications-not-made_{context}"]
+== Operating system file system modifications not made with the Machine Config Operator
+
+No assurances are made at this time that a modification made to the host operating file system is preserved across minor releases except for where that modification is made through the public interface exposed with the Machine Config Operator.
```

Comment:
and augment this to say "modification is made through the public interface exposed via a supported operator (Machine Config Operator, Node Tuning Operator, etc.)." to reflect there are a cross-section of operators that do in fact touch the host, and the list is not as simple as when this was first written.